tester/orb.yaml

```diff
@@ -162,11 +163,28 @@ jobs:
         description: >
           Docker-compose args such as a chain of files to use.
         type: string
+      creds:
+        default: GCLOUD_SERVICE_KEY
+        description: >
+          Name of environment variable storing the base64-encoded service key
+          for the GCP project.
+        type: env_var_name
       executor:
         default: cimg/base:2020.01
         description: >
           Name of the docker image to use to execute the job.
         type: string
+      user:
+        # this parameter exists because base executor image defaults to non-root user
+        default: root
```

Comment:
note: this means we'll need to push out a breaking change, since some folks may have been relying on a non-root user default